.circleci/config.yml

```diff
@@ -16,32 +16,44 @@ workflows:
               ignore: /.*/
 
 jobs:
-  test: # runs not using Workflows must have a `build` job as entry point
+  test:
+    resource_class: xlarge
     docker:
       # CircleCI Go images available at: https://hub.docker.com/r/circleci/golang/
-      - image: circleci/golang:1.14-buster
+      - image: circleci/golang:1.13
 
     steps: # steps that comprise the `build` job
       - checkout
 
-      - restore_cache: # restores saved cache if no changes are detected since last run
+      # restores saved cache if no changes are detected since last run
       # Read about caching dependencies: https://circleci.com/docs/2.0/caching/
+      - restore_cache:
           keys:
-            - v1-pkg-cache
+            - pkg-cache
+
+      - restore_cache:
+          keys:
+            - testbin-cache
```

Comment:
envtest tests require etcd and kupe-apiserver binaries. Cache them to avoid downloading them on each run.